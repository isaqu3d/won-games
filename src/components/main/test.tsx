import { render, screen } from '@testing-library/react'
import Main from '.'

describe('<Main />', () => {
  it('should render the heading', () => {
    const { container } = render(
      <Main
        title="React Avançado"
        description="TypeScript, ReactJS, NextJS e TailwindCSS"
      />
    )

    expect(
      screen.getByRole('heading', {
        name: /react avançado/i
      })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
