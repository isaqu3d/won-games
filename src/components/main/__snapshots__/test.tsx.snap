// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<main
  class="bg-[#06092b] text-white text-lg h-full font-sans p-12 flex items-center justify-center text-center flex-col"
>
  <img
    alt="Imagem de um átomo e React Avançado escrito ao lado."
    class="w-80 mb-8"
    src="/img/logo.svg"
  />
  <h1>
    React Avançado
  </h1>
  <p>
    TypeScript, ReactJS, NextJS e TailwindCSS
  </p>
  <img
    alt="Um desenvolvedor de frente para uma tela com código."
    class="w-full max-w-[30rem] mt-12"
    src="/img/hero-illustration.svg"
  />
</main>
`;
